<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateNewLink</class>
 <widget class="QDialog" name="CreateNewLink">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
  color:black;
   font-family:&quot;Microsoft YaHei&quot;;
}
QLineEdit#linkNameLine
{
border:1px solid blue;
background: transparent
}
QRadioButton::indicator
{
width:28px;
height:28px;
}
QRadioButton::indicator::unchecked
{
background: transparent;
border:1px white;

}
QRadioButton::indicator::checked
{
border-image: url(:/new/image/FastCombatImage/TeamConfigImage/multiSelect.png);
}
/*设置QListWidget背景与item样式*/
QListWidget#protoListW
{
background-image: url(:/LinkMImage/listWBgk.png);
/*border-left:3px solid white;
border-bottom:3px solid white;*/
}
QListWidget::item
{
background: transparent;
}
QListWidget::item::hover
{
background-color: rgb(85, 85, 255);
}
QListWidget::item::selected
{
background-color: rgb(85, 170, 255);
}
/*水平纯色滚动条*/
QScrollBar:horizontal {/*横向背景色*/
      border: 1px solid grey;
      background: transparent;
      height: 10px;
      margin: 0px 0px 0 0px;
  }
QScrollBar::handle:horizontal {/*小滑块背景色*/
      background: rgb(85, 85, 255);
      min-width: 5px;
	border-radius:2px
}
/*水平纯色滚动条*/
 QScrollBar:vertical {/*横向背景色*/
      border: 1px solid grey;
      background: transparent;
      height: 10px;
      margin: 0px 0px 0 0px;
  }
QScrollBar::handle:vertical {/*小滑块背景色*/
      background: rgb(85, 85, 255);
      min-width: 5px;
}
/*两段滚动条*/
QScrollBar::add-page,QScrollBar::sub-page{
   background: transparent;}
QScrollBar::add-line,QScrollBar::sub-line{
   background: transparent;}

/*下拉框背景色选择*/
QComboBox#linkTypeCom,#linkFunCom
{
background-color: rgb(85, 85, 255);
border:1px solid white;
border-radius:3px;
}
QComboBox#linkTypeCom QAbstractItemView
{
background-color: rgb(85, 170, 255);
selection-background-color:green;

}
QComboBox#linkFunCom QAbstractItemView
{
background-color: rgb(85, 170, 255);
selection-background-color:green;

}</string>
  </property>
  <widget class="QGroupBox" name="generalGroup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>681</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>General</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>连接名称：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="linkNameLine">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>连接类型：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>连接功能：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="linkTypeCom">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Serial</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TCP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>UDP组播</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>UDP单播</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>UDP广播</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="linkFunCom">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>仅用于发送</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>仅用于接收</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>即发送又接受</string>
     </property>
    </item>
   </widget>
   <widget class="QListWidget" name="protoListW">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="autoScrollMargin">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Link1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Link2</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>连接状态：</string>
    </property>
   </widget>
   <widget class="QLabel" name="conStausLab">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="autoConcheck">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>创建连接后自动连接</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancleButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>530</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>530</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="proSetGroup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>700</width>
     <height>200</height>
    </rect>
   </property>
   <property name="title">
    <string>串口-仅用于发送-配置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>14</width>
     <height>14</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>14</width>
     <height>14</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>createNewLink</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
